app/src/main/java/ch/epfl/sdp/lobby/PlayerParametersFragment.kt

```diff
@@ -0,0 +1,83 @@
+package ch.epfl.sdp.lobby
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.CompoundButton
+
+import ch.epfl.sdp.R
+import ch.epfl.sdp.databinding.FragmentPlayerParametersBinding
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [PlayerParametersFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class PlayerParametersFragment : Fragment() {
+
+    enum class Faction {
```

Comment:
Maybe we should pull the enum out on a class of it's own since it might be used by multiple components